lms/djangoapps/courseware/access.py

```diff
@@ -766,6 +721,25 @@ def _has_access_to_course(user, access_level, course_key):
     return ACCESS_DENIED
 
 
+def adminstrative_accesses_to_course_for_user(user, course_key):
+    """
+    Returns types of access a user have for given course.
+    """
+    global_staff = GlobalStaff().has_user(user)
+
+    staff_access = (
+        CourseStaffRole(course_key).has_user(user) or
+        OrgStaffRole(course_key.org).has_user(user)
+    )
+
+    instructor_access = (
+        CourseInstructorRole(course_key).has_user(user) or
+        OrgInstructorRole(course_key.org).has_user(user)
+    )
+
+    return global_staff, staff_access, instructor_access
```

Comment:
@adampalay I met with @noraiz-anwar and made things clearer and now i think we don't need a review from platform.